security_monkey/__init__.py

```diff
@@ -213,13 +214,33 @@ def send_email(msg):
 
 # Logging
 import sys
-from logging import Formatter
+from logging import Formatter, handlers
```

Comment:
Why are we importing `handlers` ?  The next line imports `logging.handlers.RotatingFileHandler`.  Seems like in the `GroupWriteRotatingFileHandler` class, we could simply reference `RotatingFileHandler` instead of `handlers.RotatingFileHandler`.